libraries/app/application.cpp

```diff
@@ -947,9 +947,9 @@ void application::set_program_options(boost::program_options::options_descriptio
          ("enable-standby-votes-tracking", bpo::value<bool>()->implicit_value(true),
           "Whether to enable tracking of votes of standby witnesses and committee members. "
           "Set it to true to provide accurate data to API clients, set to false for slightly better performance.")
-         ("plugins", bpo::value<string>(), "Space-separated list of plugins to activate")
          ("accept-incoming-connections", bpo::value<bool>()->implicit_value(true), "Accept incoming connections")
-         ("advertise-peer-algorithm", bpo::value<string>()->implicit_value("all"), "Determines which peers are advertised")
+         ("advertise-peer-algorithm", bpo::value<string>()->implicit_value("all"), "Determines which peers are advertised. "
+            "Options are: \"random\", \"list\", or \"nothing\"")
```

Comment:
... and "all". BTW the line above is a little long.

Reply:
Done